examples/platforms/nrf52840/README.md

```diff
@@ -110,9 +122,7 @@ $ make -f examples/Makefile-nrf52840 DISABLE_CC310=1
 ## Flashing the binaries
```

Comment:
Edit for line 115:

Optional disabling of CryptoCell 310 support

The previous header is "optional", and I assume this one is too, so let's make it clear this is not mandatory.